translations.yml

```diff
@@ -430,7 +430,7 @@
   uk: 'Нова задача на GitHub'
   ar: 'قضية جديدة على Github'
   cs: 'Nová GitHub Issue'
-  ja: '新しいGitHubの課題'
+  ja: 'GitHubで新しいIssueを作成する'
```

Comment:
Issue is famous word in Japan too.

It is written in Japanese on GitHub help pages:
https://docs.github.com/ja/issues/tracking-your-work-with-issues/creating-an-issue